### PR TITLE
Initial draft of App v2 CLI support

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Adds info on using App V2 capable CLI to create and manage applications.

Also, code snippets don't need beta anymore.

## Review

Review app will be available.

### CODE REVIEW

File: _documentation/application/nexmo-cli.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+## Updating an application
+
+You can update a previously created application with a command similar to:
+
+```
+nexmo app:update asdasdas-asdd-2344-2344-asdasda12345 "Updated Application" --capabilities=voice,rtc --voice-event-url=http://example.com/webhooks/event --voice-answer-url=http://example.com/webhooks/answer --rtc-event-url=http://example.com/webhooks/rtcevent
+```
+
+You can change the application name, modify any of the webhooks, or add new capabilities.
+
```
[the remark this answers is not in the document]
Comment (tbedford) [reply _documentation/application/nexmo-cli.md]: Yes! :)

**Comment by tbedford** [comment]
@AlexLakatos if you are happy with the latest additions pls feel free to merge! Thanks!

File: _documentation/application/overview.md [hunk 2]
```diff
@@ -71,5 +71,19 @@ There are four main ways to create an application:
 
 1. In the Nexmo Dashboard you can create an application with a single capability. You can then add capabilities and update the details of an application created in the Dashboard using the v2 API. If an application has multiple capabilities it will appear in all relevant sections of the Dashboard. For example, an application with both Voice and Messages capabilities will appear in both the Voice and Messages sections of the Dashboard.
 2. Using the Nexmo CLI (this is currently restricted to v1 Applications with a single fixed capability).
-3. Using the [Applications API](https://developer.nexmo.com/api/application.v2).
+3. Using the [Application API](https://developer.nexmo.com/api/application.v2).
 4. Using one of the Nexmo client libraries (this is currently restricted to v1 Applications with a single fixed capability).
+
+## Managing applications using the CLI
+
+* [Managing application suing the Nexmo CLI](/application/nexmo-cli)
```
Comment (AlexLakatos) [inline _documentation/application/overview.md]:
```suggestion
* [Managing application using the Nexmo CLI](/application/nexmo-cli)
```

**Comment by AlexLakatos** [comment]
Looks good, merging as soon as the new review app is done.

**Comment by tbedford** [comment]
@AlexLakatos sorry I just saw a last minute typo - now fixed.

**Comment by AlexLakatos** [comment]
I saw the same thing. that's why I was waiting for the review app. :) https://github.com/Nexmo/nexmo-developer/pull/2004/commits/7a3499321df8403f33af0af346c7411086e44e35

**Pull request merged**